Source: http://osdir.com/ml/fedora-extras-commits/2009-03/msg00924.html
Upstream: Should be in QT 4.5.1
Reason: Fix a compile issue on ppc64

--- qt-x11-opensource-src-4.5.0/src/corelib/arch/qatomic_powerpc.h.orig	2009-02-25 22:09:21.000000000 +0100
+++ qt-x11-opensource-src-4.5.0/src/corelib/arch/qatomic_powerpc.h	2009-03-17 14:17:25.000000000 +0100
@@ -193,7 +193,7 @@
                  "lwarx  %[result]," _Q_VALUE "\n"
                  "xor.   %[result], %[result], %[expectedValue]\n"
                  "bne    $+12\n"
-                 "stwcx. %[newValue],0,%[_q_value]\n"
+                 "stwcx. %[newValue]," _Q_VALUE "\n"
                  "bne-   $-16\n"
                  : [result] "=&r" (result),
                    _Q_VALUE_MEMORY_OPERAND
